.github/workflows/pr-test-e2e.yml

```diff
@@ -47,13 +47,14 @@ jobs:
           tar -xvf kubeflex_0.4.2_linux_amd64.tar.gz bin/kflex
           mv bin/kflex /usr/local/bin
           rm -fr bin kubeflex_0.4.2_linux_amd64.tar.gz
+          go install github.com/onsi/ginkgo/v2/ginkgo
 
       - name: Run test
         env:
           TEST_FLAGS: ${{ github.event.inputs.testFlags }}
         run: |
-          cd test/e2e/multi-cluster-deployment 
-          KFLEX_DISABLE_CHATTY=true ./run-test.sh $TEST_FLAGS
+          cd test/e2e/ginkgo 
+          KFLEX_DISABLE_CHATTY=true KUBECONFIG=~/.kube/config ginkgo -v $TEST_FLAGS
```

Comment:
It looks strange to set `KUBECONFIG` to the value that has the same meaning as leaving it unset. Why is that there?

Reply:
It looks like ginkgo is somehow setting a different default value of KUBECONFIG.  If I don't set the KUBECONFIG, then the setup script fails when it first attempts to access the kind-kubeflex context.  I'll dig further into it.

Reply:
Any news on this?